packages/@aws-cdk/pipelines/lib/legacy/actions/publish-assets-action.ts

```diff
@@ -99,6 +99,13 @@ export interface PublishAssetsActionProps {
    * @default -
    */
   readonly preInstallCommands?: string[];
+
+  /**
+   * Stage name where action is used in.
+   *
+   * @default Assets
+   */
+  readonly stageName?: string;
```

Comment:
Considering that this is always being set to `Assets` in the pipeline package, I'm unclear what the use case would be for this to be defined as something else here.